doc/source/serve/http-guide.md

```diff
@@ -74,6 +74,51 @@ Existing middlewares, **automatic OpenAPI documentation generation**, and other
 Serve currently does not support WebSockets. If you have a use case that requires it, please [let us know](https://github.com/ray-project/ray/issues/new/choose)!
 ```
 
+(serve-http-streaming-response)=
```

Comment:
Should we pull this in a separate page for discoverability (or rename this to something better than HTTP Guide)?

Reply:
We already have a lot of user guides and this feels like the natural place for it (where we explain starlette & fastapi behaviors)

Reply:
no preference on the naming if you have a better one suggest it

Reply:
Once we add batching + a real-world example we can have a separate page for that. Not enough time before branch cut for those though

Reply:
That sounds good, we have more time for docs changes btw, so we can do it in the follow-up PR.

Reply:
Ah ok I'll work on it first thing next week then